### PR TITLE
Enhance document and add the miss steps.

### LINKED ISSUE #19: Enhance document for building container image and packages for local development
**What would you like to be added**:

I would like to enhance the existing document to include instructions for building container images locally and generating packages using the container image.

**Why is this needed**:

This update is necessary because the current document lacks certain steps and details required for developers to successfully build container images and generate packages. By providing comprehensive instructions, developers will have a clearer understanding of the process and can effectively build container images and generate packages for local development purposes.

### PR DESCRIPTION
**Pull request by alee-cd**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #19 

**Special notes for your reviewer**:

**Release note**:

```feature user

```

### CODE REVIEW

**Comment by ghost** [comment]
@alee-ntap Thank you for your contribution.

File: README.md [hunk 1]
```diff
@@ -12,25 +12,54 @@ It contains:
...
 
 ### Build container
 
-on amd64:
+To build the container, follow the steps below:
+
+Install the `podman` package:
+```
+apt install podman
+```
```
Comment (Vincinator) [inline README.md]:
```suggestion
Install `podman` on your system: https://podman.io/docs/installation
```

**Comment by Vincinator** [comment]
Thanks @alee-ntap looks good to me. 👍🏻  

Added a small suggestion to reference the podman install guide, instead of debian specific install cmd

File: README.md (hunk 1 above)
Comment (alee-cd) [reply README.md]: Added your changes.

**Review by nkraetzschmar — APPROVED** [review]
lgtm

**Pull request merged**